src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2403,7 +2403,7 @@ println "Unlocking Task B"
 synchronizationapi.put(variables.get("PA_JOB_ID"), "lock", false)
 ----
 
- * *Clean*: Delete the channel. This task simply deletes the channel used in this job. As there is no automatic mechanism to remove channels, it is necessary to delete them explicitely when they are not used any more.
+ * *Clean*: Delete the channel. This task simply deletes the channel used in this job. As there is no automatic mechanism to remove channels, it is necessary to delete them explicitly when they are not used anymore.
```

Comment:
The default syntax is "any more". "Anymore" is possible, according to:
"Especially in American English, any more, as an adverb, can be written as one word, anymore:

He doesn’t cycle anymore."

Reply:
I did not know that, Good to know. Thanks!